tests/Jellyfin.Api.Tests/Jellyfin.Api.Tests.csproj

```diff
@@ -37,6 +37,7 @@
   <ItemGroup>
     <ProjectReference Include="../../Jellyfin.Api/Jellyfin.Api.csproj" />
     <ProjectReference Include="../../Jellyfin.Server.Implementations/Jellyfin.Server.Implementations.csproj" />
+    <ProjectReference Include="..\Jellyfin.Server.Integration.Tests\Jellyfin.Server.Integration.Tests.csproj" />
```

Comment:
Stay consistent and use forwards slashes everywhere

Reply:
That'll be Visual Studio running on windows.